.github/workflows/ci.yml

```diff
@@ -198,7 +198,7 @@ jobs:
         run: |
           pip3 install Pillow
           pip3 install Wave
-          pip3 install numpy
+        run: pip3 install numpy
```

Comment:
is this a typo? was this change needed?

Reply:
I think this was from a previous merge.  I shouldn't be making any changes here now.  I'll revert ci.yaml to the current upstream.